src/tests/stdlib/wcstombs_test.c

```diff
@@ -0,0 +1,50 @@
+/**
+ * @file
+ * @brief Test unit for stdlib/wcstombs and stdlib/wctomb.
+ *
+ * @date Nov 16, 2018
+ * @author Chubukov Filipp
+ */
+
+#include <embox/test.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+
+#define TEST_STRING L"ABC"
+#define CHECK_STRING "ABC"
+
+EMBOX_TEST_SUITE("stdlib/wcstombs_test and stdlib/wctomb");
+
+TEST_CASE("wcstombs compare ABC") {
+	int length = 0;
+	char check_string[] = CHECK_STRING;
+	wchar_t test_string[] = TEST_STRING;
+	char buffer[10];
+
+	length = wcstombs(buffer, test_string, sizeof(check_string));
+	test_assert(length == sizeof(check_string) - 1);
+	test_assert(memcmp(buffer,check_string,length) == 0);
+}
+
+TEST_CASE("wctomb compare ABC") {
+	int length = 0;
+	char check_string[] = "A";/* Just first symbol of test_string */
+	wchar_t test_string[] = TEST_STRING;
+	char buffer[10];
```

Comment:
Why 10 bytes?

Reply:
stock in case functions do something wrong